biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -1169,12 +1196,12 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
               
               ! Compare with Ball-Berry model: gs_mol = m * an * hs/leaf_co2_ppress p + b
               hs = (gb_mol*ceair + gs_mol* veg_esat ) / ((gb_mol+gs_mol)*veg_esat )
-              gs_mol_err = bb_slope(ft)*max(anet, 0._r8)*hs/leaf_co2_ppress*can_press + bbb
+              gs_mol_err = bb_slope(ft)*max(anet, 0._r8)*hs/leaf_co2_ppress*can_press + stomatal_intercept_btran
               
-              if (abs(gs_mol-gs_mol_err) > 1.e-01_r8) then
-                 write (fates_log(),*) 'CF: Ball-Berry error check - stomatal conductance error:'
-                 write (fates_log(),*) gs_mol, gs_mol_err
-              end if
+              if (abs(gs_mol-gs_mol_err) > 1.e-01_r8 .and.  (stomatal_model == 1)) then
```

Comment:
@Qianyuxuan @rgknox this part. It used to be specific to ball berry but maybe this is more general and should instead say something like "Stomatal model error check - stomatal conductance error:" or similar

I assume this is a relevant check for any stomatal model yes?

Reply:
Sorry. I was thinking about it but then hesitated to include it. I found CLM5 didn't code the logging of Medlyn model error. I can definitely include those codes if it is a common procedure.

Reply:
@Qianyuxuan OK.  @rgknox @glemieux this looks like a basic error/mismatch calculation, any reason to think it cant be generic to both models?  or maybe I am missing something?

Reply:
I think your right, I'd have to look more closely, but yup, I think that convergence error is used in both schemes

Reply:
Okay. I will add that to my next commit!